app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -814,15 +817,23 @@ void OpenXRInputSource::Update(const XrFrameState& frameState, XrSpace localSpac
     bool usingTrackedPointer = pointerMode == DeviceDelegate::PointerMode::TRACKED_POINTER;
     bool hasAim = isPoseActive && (poseLocation.locationFlags & XR_SPACE_LOCATION_ORIENTATION_VALID_BIT);
 
+
+    if (hasAim && mUsingHandInteractionProfile && handFacesHead) {
```

Comment:
mUsingHandInteracton is true also for devices where only the EXT_HAND_ITERACTION extension is supported; perhaps setting hasAim to false doesn't affect in that case, but the 'if' clause code is confusing; shouldn't we just check for the MSFT extension, as the comment says ?

Reply:
Yeah I though about that but decided to leave it like this because it was safer, in the sense that we will be "protected" against changes in the runtimes. I can change it if you want to check for the MSFT extension or reword the comment, as you prefer.

Reply:
I think rewording the comment, including additional information about what you mentioned about protect against runtieme changes, would be useful.